dom/src/test/java/org/apache/james/mime4j/field/datetime/DateTimeTest.java

```diff
@@ -101,10 +103,9 @@ public void testNormalDate() throws ParseException {
 
     private void ensureAllEqual(String[] dateStrings) throws ParseException {
         for (int i = 0; i < dateStrings.length - 1; i++) {
-            Assert.assertEquals(
-                    new DateTimeParser(new StringReader(dateStrings[i])).parseAll().getDate().getTime(),
-                    new DateTimeParser(new StringReader(dateStrings[i + 1])).parseAll().getDate().getTime()
-            );
+            assertThat(new DateTimeParser(new StringReader(dateStrings[i])).parseAll().getDate().getTime())
```

Comment:
It's quite difficult for me to understand what is equals to what. Could we separate realValue and expectedValue?